### DISCUSSION TITLE
Nginx Config

### DISCUSSION THREAD

**Discussion by reset12** [pH7Software/pH7-Social-Dating-CMS#853] [Troubleshooting]
Hello,
is there a running NGINX Config with the converted .htaccess ?
With the converter from Internet i get errors.

with regards
reset12

**Comment by polynamaude** [comment]
@reset12 Why don't you try converting yourself the .htacess file for Nginx ?
You'll need to customize it anyway, so you'll learn now or tomorrow.

**Comment by reset12** [comment]
I think it is not necessary  to reinvent the wheel and it will be helpfull put that nginx-config to the documentation.

**Comment by polynamaude** [comment]
@reset12 I think it's needed to know how sofware work before trying to use it.
As people won't answer all the question and become tech support for your own project.

**Comment by polynamaude** [comment]
I really don't know what this file is used for
https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/master/nginx.conf

Also, there's already documentation for Nginx.
https://ph7cms.com/doc/en/nginx-configuration

**Comment by reset12** [comment]
Sorry but the documentation for nginx is not uptodate an usable.
I have installed Nextcloud on Nginx on my private server, but this documentation is only a little part of the nginx-conf.
I am not be able to login with the admin123 because the apache rewrite rules are not includes in the nginx-config.

Software should be installable without read and understand all the code, only with the installation manual.
When someone have installed ph7cms on Nginx it should be possible to get the complete Config in the manual.
I think it is a good project and i like it to try for a private free site to find people in the neighborhood with same Interests and Hobby after Corona.

**Comment by polynamaude** [comment]
Great for all your expectation (even more with something free). Your contribution will probably be accepted by the project owner if you feel like contributing.

Maybe you should make a difference between software meant to be used by professional and other made for use by layman. This is not microsoft Office here.

Good luck.
